charts/fluid/fluid/templates/controller/vineyardruntime_controller.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: vineyardruntime-controller
+  namespace: {{ include "fluid.namespace" . }}
+  labels:
+    control-plane: vineyardruntime-controller
+spec:
+  selector:
+    matchLabels:
+      control-plane: vineyardruntime-controller
+  {{ if .Values.runtime.vineyard.enabled -}}
+  replicas: {{ .Values.runtime.vineyard.replicas }}
+  {{- else }}
+  replicas: 0
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        control-plane: vineyardruntime-controller
+      annotations:
+      {{ if gt (.Values.runtime.vineyard.replicas | int) 1 -}}
+        controller.runtime.fluid.io/replicas: {{ .Values.runtime.vineyard.replicas | quote }}
+      {{- end }}
+    spec:
+      {{- with .Values.image.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: vineyardruntime-controller
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: type
+                operator: NotIn
+                values:
+                - virtual-kubelet
+      {{- if .Values.runtime.vineyard.tolerations }}
+      tolerations:
+{{ toYaml .Values.runtime.vineyard.tolerations | indent 6 }}
+      {{- end }}
+      containers:
+      - image: {{ .Values.runtime.vineyard.controller.image | quote }}
+        name: manager
+        command: ["vineyardruntime-controller", "start"]
+        args:
+          - --development=true
```

Comment:
How about setting development to false for production environment?

Reply:
Make sense to me.